### PR TITLE
Calc event

### PR DESCRIPTION
**Pull request by MahdiRezaeiDev**
### Math magicians: build Calculator functionality
In this branch, I have made the following changes:

- Added the `calculate.tsx` file to perform the screen rendering operation.
- Added the `operate.tsx` file to perform mathematical operations.
- Added state to the `Calculator.tsx` file to do math operations and update DOM.

### CODE REVIEW

File: src/components/Calculator.tsx [hunk 1]
```diff
@@ -1,34 +1,59 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
 import React from 'react';
 import Screen from './Screen';
 import Key from './Key';
-
+import calculate from '../logic/calculate';
```
Comment (mateo951) [inline src/components/Calculator.tsx:5]: The `Calculator` component imports the `calculate.js` file from the logic directory ✔

File: src/components/Calculator.tsx [hunk 2]
```diff
@@ -1,34 +1,59 @@
...
-          <Key area="two" keyValue="2" />
-          <Key area="three" keyValue="3" />
-          <Key area="subtract" keyValue="-" />
-          <Key area="dot" keyValue="." />
-          <Key area="zero" keyValue="0" />
-          <Key area="divide" keyValue="/" />
-          <Key area="multiple" keyValue="*" />
-          <Key area="reset" keyValue="RESET" />
-          <Key area="equal" keyValue="=" />
+          {keys}
```
Comment (mateo951) [inline src/components/Calculator.tsx:56]: Nice thinking! Implement all calculator operations ✔

**Review by mateo951 — APPROVED** [review]
### Hi @MahdiSohaily, 

**🔥 Highlights 🔥**
- Props need gets passed to child components of the `Calculate.js` component ✔

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.tsx (hunk 2 above)
Comment (MahdiRezaeiDev) [reply src/components/Calculator.tsx:56]: Thank you dear @mateo951.
It was the first time that I feel the code reviewer really read my code and provided feedback.
all other feedbacks tell now was like a response from a robot.

**Pull request merged**